UI/obs-app.cpp

```diff
@@ -3445,6 +3459,10 @@ int main(int argc, char *argv[])
 	log_blocked_dlls();
 #endif
 
+#ifndef _WIN32
+	bfree(ss.ss_sp);
```

Comment:
Is it safe to release this pointer though the stack can be used?
Maybe it's better to revert the signal stack context before releasing `ss.ss_sp`.

Reply:
Yeah, sounds like a reasonable idea. Thanks.